src/main/app/src/components/laskuri/KeskiarvoTulos.tsx

```diff
@@ -105,55 +121,59 @@ export const KeskiarvoTulos = ({ tulos }: Props) => {
   const { t } = useTranslation();
   return (
     <TulosContainer>
-      <Typography variant="h3" sx={{ fontSize: '1.25rem' }}>
-        {t('pistelaskuri.lukio.header')}
-      </Typography>
-      <Typography variant="h3" sx={{ fontSize: '1.25rem' }}>
-        {t('pistelaskuri.ammatillinen.header')}
-      </Typography>
-      <ResultSphere
-        result={tulos.keskiarvo}
-        text={t('pistelaskuri.pisteet.lukio')}></ResultSphere>
-      <Box className={classes.spheresContainer}>
-        <ResultSphere
-          result={tulos.pisteet + 2}
-          text={t('pistelaskuri.pisteet.ammatillinen-first')}></ResultSphere>
-        <ResultSphere
-          result={tulos.pisteet}
-          text={t('pistelaskuri.pisteet.ammatillinen-rest')}></ResultSphere>
-      </Box>
-      <Paper className={classes.textContainer} elevation={0}>
-        <Typography
-          variant="body1"
-          className={classes.textBlock}
-          sx={{ marginBottom: '0.5rem' }}>
-          {t('pistelaskuri.lukio.body')}
+      <Box className={classes.column}>
+        <Typography variant="h3" sx={{ fontSize: '1.25rem' }}>
+          {t('pistelaskuri.lukio.header')}
         </Typography>
-        <LinkToValintaPerusteet></LinkToValintaPerusteet>
-      </Paper>
-      <Paper className={classes.textContainer} elevation={0}>
-        <Typography variant="body1" className={classes.textBlock}>
-          {t('pistelaskuri.ammatillinen.huomioitu')}
-        </Typography>
-        <List sx={{ fontSize: '0.875rem', listStyleType: 'disc', pl: 2 }} dense={true}>
-          <ListItem sx={{ display: 'list-item' }}>
-            {t('pistelaskuri.ammatillinen.pisteytys-yleinen')}
-          </ListItem>
-          <ListItem sx={{ display: 'list-item' }}>
-            {t('pistelaskuri.ammatillinen.pisteytys-painotettava')}
-          </ListItem>
-          <ListItem sx={{ display: 'list-item' }}>
-            {t('pistelaskuri.ammatillinen.pisteytys-ensisijainen')}
-          </ListItem>
-        </List>
-        <Typography
-          variant="body1"
-          className={classes.textBlock}
-          sx={{ marginBottom: '0.5rem' }}>
-          {t('pistelaskuri.ammatillinen.oletus')}
+        <ResultSphere result={tulos.keskiarvo} text={t('pistelaskuri.pisteet.lukio')} />
+        <Paper className={classes.textContainer} elevation={0}>
+          <Typography
+            variant="body1"
+            className={classes.textBlock}
+            sx={{ marginBottom: '0.5rem' }}>
+            {t('pistelaskuri.lukio.body')}
+          </Typography>
+          <LinkToValintaPerusteet />
+        </Paper>
+      </Box>
+      <Box className={classes.column}>
+        <Typography variant="h3" sx={{ fontSize: '1.25rem' }}>
```

Comment:
Olis tosi hyvä, jos käytettäis nimettyjä typografia-variantteja, ettei nää enempää ränsistyis ja pysyis fonttien tyyli yhdenmukaisena. Muuten meillä on kohta sata erilaista typografiaa, kun joka paikkaan laitetaan inline-fonttikoko ja muut määreet :sweat_smile: